xclim/indices/_threshold.py

```diff
@@ -1365,3 +1366,74 @@ def tropical_nights(
     return (
         tasmin.pipe(lambda x: (tasmin > thresh) * 1).resample(time=freq).sum(dim="time")
     )
+
+
+@declare_units("", tas="[temperature]", thresh="[temperature]", sum_thresh="K days")
+def degree_days_depassment_date(
+    tas: xarray.DataArray,
+    thresh: str,
+    sum_thresh: str,
+    op: str = ">",
+    after_date: str = None,
+    freq: str = "YS",
+):
+    r"""Degree days depassment date.
+
+    Day of year when the sum of degree days exceeds a threshold. Degree days are
+    computed above or below a given temperature threshold.
+
+    Parameters
+    ----------
+    tas : xarray.DataArray
+      Mean daily temperature.
+    thresh : str
+      Threshold temperature on which to base degree days evaluation.
+    sum_thresh : str
+      Threshold of the degree days sum, in "K days".
+    op : {">", "gt", "<", "lt", ">=", "ge", "<=", "le"}
+      If equivalent to '>', degree days are computed as `tas - thresh` and if
+      equivalent to '<', they are computed as `thresh - tas`.
+    after_date: str, optional
```

Comment:
Modify after_date definition to 'start date of degree day accumulation - default is the beginning of resample period' or similar